Dockerfile

```diff
@@ -1,10 +1,9 @@
 FROM openjdk:11.0.7-jdk
 
-ARG VERSION
-ARG DATADOG_VERSION="0.49.0"
-ENV APP_VERSION=${VERSION}
+LABEL version="1.0.0" description="Guiabolso Connector" maintainer="Guiabolso Connect<suporteconnect@guiabolso.com.br>"
 
-LABEL maintainer="Guiabolso Team"
+ARG DATADOG_VERSION="0.49.0"
+ENV APP_VERSION="1.0.0"
```

Comment:
Essas mudanças aqui deixa o docker file menos flexível, mantenha a variável `VERSION` pois esse valor vai mudar quando o app for evoluindo. Ainda, tem um script que pretendemos usar para fazer release do app no dockerhub que quebraria com essas mudanças, [release.sh](https://github.com/GuiaBolso/guiabolso-connector/blob/master/release.sh)